spec/mutations/team_activate_spec.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Team Activate', type: :request do
+  include AuthHelper
+  include JsonHelper
+
+  def query(team_id:)
+    %(
+      mutation {
+        teamActivate(input:{
+          teamId: "#{team_id}"
+        }) {
+          errors
+        }
+      }
+    )
+  end
+
+  let(:team) { create(:team) }
+  let(:current_user) { create(:user, teams: [team]) }
+
+  describe 'success' do
+    it 'updates the active team for the user' do
```

Comment:
It's been a while. My context of what a "team" means is fuzzy. What is the reason we want to make sure there is only one "active team" at a time?

Reply:
Teams are a new concept per some of our new thinking about [the kind of jobs](https://hbr.org/ideacast/2016/12/the-jobs-to-be-done-theory-of-innovation.html) that folks might use his product for:  namely to connect more closely with people that they work with.

The basic thinking is that a team of folks might subscribe to use the service -- but that a person might themselves belong to more than one team (think Slack).  We want to know what team is "active" so that we can show the user that team's corresponding rooms and let them join, sort of like [this frontend pr](https://github.com/go-between/musicbox/pull/3) starts to show.

I owe some more posting about this and what we need to do next in basecamp -- maybe tomorrow evening.